spec/integration/init_web_spec.cr

```diff
@@ -106,20 +119,6 @@ describe "Initializing a new web project" do
     message = "Projects cannot be named app, app_database, app_server, shards, start_server."
     output.to_s.strip.should contain(message)
   end
-
-  it "translates dashes to underscores in the project name" do
-    output = IO::Memory.new
-    Process.run(
-      "crystal run src/lucky.cr -- init.custom test-project",
-      env: ENV.to_h,
-      output: output,
-      shell: true
-    )
-
-    shard_yml_name = File.read_lines("test-project/shard.yml").find { |line| line =~ /^name:/ }
-    shard_yml_name.should eq("name: test_project")
-    File.exists?("test-project/src/test_project.cr").should be_true
```

Comment:
This is still tested, it's just moved to the first spec block. These specs are pretty expensive to run, and since all of the integration specs use the same `test-project`, we can piggyback off the first spec to ensure the name is translated from dashes to underscores, and save some time by generating 1 less project.